setup.cfg

```diff
@@ -33,25 +33,6 @@ source = signac
 omit = 
 	*/signac/common/configobj/*.py
 
-[tool:pytest]
-filterwarnings = 
-	error::FutureWarning:signac.*
-	error::DeprecationWarning:signac.*
```

Comment:
We should leave these warnings as errors. That's how we know we've removed them all. The only lines that should be deleted are the `ignore:` lines.

Reply:
Whoops thanks I missed that these weren't all ignores in the batch delete.